index.html

```diff
@@ -642,12 +646,25 @@
       </p>
       <pre class="idl">
         interface PushManager {
+          [SameObject] static readonly attribute FrozenArray&lt;DOMString&gt; supportedContentEncodings;
```

Comment:
Hmm, it's called a content coding, but the header is content-encoding.  I think that `supportedContentEncodings` is probably the best choice, because other than us nerds who work on this stuff, the difference is meaningless.